contracts/StreamManager.sol

```diff
@@ -42,6 +42,11 @@ contract StreamManager is IStreamManager, ReentrancyGuard {
      * @param _payer new address  
      */
     event PayerChanging(address _payer);
+    /**
+     * @dev Changing address of fee
+     * @param _feeAddress new address of the fee
+     */
+    event AddressFeeChanging(address _feeAddress); 
```

Comment:
@CTAPCKPIM please use past tense for the event names. And update accordingly. Thanks!